test/AppointmentForm.test.js

```diff
@@ -1,42 +1,74 @@
 import React from 'react';
-import ReactTestUtils from 'react-dom/test-utils';
+import 'whatwg-fetch';
+import ReactTestUtils, { act } from 'react-dom/test-utils';
 import { createContainer } from './domManipulators';
 import { AppointmentForm } from '../src/AppointmentForm';
+import { fetchResponseOk, fetchResponseError, fetchRequestBody } from './spyHelpers';
 
 describe('AppointmentForm', () => {
-  let render; let
-    container;
+  let render;
+  let container;
+  let field;
+  let form;
+  let labelFor;
+  let change;
+  let submit;
 
   beforeEach(() => {
-    ({ render, container } = createContainer());
+    ({
+      render, container, field, form, labelFor, change, submit,
+    } = createContainer());
+    jest
+      .spyOn(window, 'fetch')
+      .mockReturnValue(fetchResponseOk({}));
   });
-  const form = (id) => container.querySelector(`form[id="${id}"]`);
+
+  afterEach = (() => {
```

Comment:
There's an extra `=` here.
It'd be good to find a way to catch errors like this. I'm not sure off the top of my head why it was allowed. Perhaps because `afterEach` is an existing mutable variable, idk.